flexinput/src/main/java/com/lytefast/flexinput/adapters/FileListAdapter.kt

```diff
@@ -34,18 +35,24 @@ class FileListAdapter(private val contentResolver: ContentResolver,
                       selectionCoordinator: SelectionCoordinator<*, Attachment<File>>)
   : RecyclerView.Adapter<FileListAdapter.ViewHolder>() {
 
+
   private val selectionCoordinator: SelectionCoordinator<*, in Attachment<File>> =
       selectionCoordinator.bind(this)
   private var files: List<Attachment<File>> = listOf()
 
+  override fun onAttachedToRecyclerView(recyclerView: RecyclerView) {
+    super.onAttachedToRecyclerView(recyclerView)
+    shrinkAnim = AnimatorInflater.loadAnimator(recyclerView.context, R.animator.selection_shrink) as AnimatorSet
+    growAnim = AnimatorInflater.loadAnimator(recyclerView.context, R.animator.selection_grow) as AnimatorSet
```

Comment:
some small cleanup is in here as well

Reply:
Are you sure there aren't going to be race conditions for this?

e.g. if VH1 tries to use it and VH2 also tries to use it? I thought these were instanced for a reason.